Description: Prefer xdg-open and x-www-browser for HTML browser.
 GRASS 6 used a shell script to set the GRASS_HTML_BROWSER environment variable.
 GRASS 7 reimplemented the shell script in Python.
 .
 The original version of this patch was made by Francesco Paolo Lovergine, and
 ported to Python by Bas Couwenberg.
Author: Francesco Paolo Lovergine <frankie@debian.org>
Author: Bas Couwenberg <sebastic@xs4all.nl>

--- a/lib/init/grass.py
+++ b/lib/init/grass.py
@@ -476,7 +476,7 @@ def set_browser():
             browser = "explorer"
         else:
             # the usual suspects
-            browsers = ["xdg-open", "htmlview", "konqueror", "mozilla",
+            browsers = ["xdg-open", "x-www-browser", "htmlview", "konqueror", "mozilla",
                         "mozilla-firefox", "firefox", "iceweasel", "opera",
                         "netscape", "dillo", "lynx", "links", "w3c"]
             for b in browsers:
